changelog.md

```diff
@@ -9,9 +9,10 @@ We expect this release to ship to DevTools in [Chrome 85](https://chromiumdash.a
 Thanks to our new contributors 👽🐷🐰🐯🐻!
 
 * Loftie Ellis @lpellis
-* Marvin Frachet @mfrachet 
-* Matt Hobbs @Nooshu 
-* Peter Marshall @psmarshall 
+* Marvin Frachet @mfrachet
+* Matt Hobbs @Nooshu
+* Peter Marshall @psmarshall
+* Umar Hansa @umaar
```

Comment:
Just happened to notice crawlable-anchors made it through! Can remove my name if that's not appropriate though

Reply:
Oh, hahaha I actually prompted your removal from this list during review since I noticed you already contributed to Lighthouse early on 😊  

but if you want to be included for your fresh start, please feel free to re-add yourself :)

Reply:
ha I see I made a rather insignificant commit here! https://github.com/GoogleChrome/lighthouse/commit/6fa6e1a8419f854d55dabdbf1a2027e6d55e5f00

I removed my name since that would unfairly surface me again! Appreciate the reminder